app/javascript/src/taxChartHelpers.js

```diff
@@ -6,42 +6,6 @@ export const nextTaxChartPosition = (game, nation) => {
 
 export const nextTaxationPowerPoints = (game, nation) => {
   nation = nation || game.currentNation;
-  let powerPoints;
-
-  if (game.baseGame === "imperial") {
-    powerPoints = nextTaxChartPosition(game, nation) - 5;
-    if (powerPoints < 0) powerPoints = 0;
-  } else if (game.baseGame === "imperial2030") {
-    const powerPointsByTax = {
-      0: 0,
-      1: 0,
-      2: 0,
-      3: 0,
-      4: 0,
-      5: 0,
-      6: 1,
-      7: 1,
-      8: 2,
-      9: 2,
-      10: 3,
-      11: 4,
-      12: 5,
-      13: 6,
-      14: 7,
-      15: 8,
-      16: 9,
-      17: 9,
-      18: 9,
-      19: 9,
-      20: 9,
-      21: 9,
-      22: 9,
-      23: 9,
-    };
-    powerPoints = powerPointsByTax[game.getTaxes(nation)];
-  }
-
-  const total = game.nations.get(nation).powerPoints + powerPoints;
-  if (total >= 25) return 25;
-  return total;
+  
+  return game.nations.get(nation).powerPoints + game.powerPointsGainedFrom(game.getTaxes(nation));
```

Comment:
I think players will get more information from this value if it isn't capped, but it could also become a source of confusion

Perhaps we should reformat the rendering of this to have the uncapped value in brackets when over 25

Reply:
Added in [77ef25a](https://github.com/Thrillberg/imperial/pull/688/commits/77ef25a69dece8cef96b54e75951c8d4c54fdb0f)
![image](https://user-images.githubusercontent.com/28380984/226118722-db4e9439-b37f-415c-866a-1c8c067420c5.png)

Reply:
👍